### ISSUE TITLE
Validate that listed HCIBox supported regions are aligned with region support

### ISSUE THREAD

**Issue by LucaOtto** [microsoft/azure_arc#2633]
**Is your issue related to a Jumpstart scenario, ArcBox, HCIBox, or Agora?**
https://azurearcjumpstart.io/azure_jumpstart_hcibox/deployment_az

**Describe the issue or the bug**
Deploying to eastus fails.

**To Reproduce**
See https://azurearcjumpstart.io/azure_jumpstart_hcibox/deployment_az

**Expected behavior**

Deployment completes

**Environment summary**
all latest version of CLI tools.

**Have you looked at the Troubleshooting and Logs section?**

**Screenshots**
Errors from the locally run command prompt and the Azure portal. 

![image](https://github.com/user-attachments/assets/1e792a9e-5a7f-42d2-b16e-6a0520e4350e)

![image](https://github.com/user-attachments/assets/35cdba9f-0a71-40f7-b96b-260edc8d704d)

**Additional context**
see issue https://github.com/microsoft/azure_arc/issues/2632 . Deploying to eastus2 doesn't work either, please advise on the best region(s) to deploy HCI Stack.

**Comment by dkirby-ms** [comment]
Try AustraliaEast

**Comment by likamrat** [comment]
@LucaOtto can i plz ask to update the issue title to something more manageable and shorter?

**Comment by WVDAdminTF** [comment]
Hi @janegilring @LucaOtto @dkirby-ms I had faced the same issue and tested in all supported regions

'australiaeast,southcentralus,eastus,westeurope,southeastasia,canadacentral,japaneast,centralindia'

Australia East is worst in terms of time, it will take 2 days for the script to complete
Other regions have other extension issues.
East US and East US 2 are currently in some form of Lock down.

Try the below two in priority
1. **Canada Central** - It will work
2. South East Asia

**Comment by LucaOtto** [comment]
Hello @dkirby-ms , @janegilring @likamrat , 

my deployment completed in Australia East and the HCI Box is up and running now.

I then added a Kubernetes cluster this way: https://azurearcjumpstart.io/azure_jumpstart_hcibox/AKS and it is connected and working.

![image](https://github.com/user-attachments/assets/22fabdb9-32d7-4db1-99a1-5e53b57d0d4b)

however I'm seeing an issue with connecting to the Kubernetes cluster , see screenshot below. I restarted the Cloud Shell multiple times, same error. 

![image](https://github.com/user-attachments/assets/77eea085-1bb1-44bb-9ba6-2f46288d3756)

I then tried to deploy a second cluster via the portal :
![image](https://github.com/user-attachments/assets/cb849393-3bd0-4553-b5a4-f9badd8f8bf2)

upgraded the cluster to the same version of the original one and tried again with the same command but I'm getting the same error. 

Any suggestions on how to fix this?
